charmhelpers/contrib/openstack/cert_utils.py

```diff
@@ -160,12 +168,12 @@ def get_certificate_request(json_encode=True, bindings=None):
                     binding,
                     hostname_override,
                     addresses)
-            # Remove hostname specific addresses from _sans
```

Comment:
This does seem to be a bug. I proved it to myself with a unit test.

Reply:
fwiw worth it is covered by this addition to the unit tests *1, without the change '10.30.0.2' is removed from the certificate request.


*1 https://github.com/juju/charm-helpers/pull/551/files#diff-facadc69ca25c0fccf4cc86904ab4e30e32088dbf213543521cc1c6bcf222ee3R136